network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/image_feature_block.html

```diff
@@ -0,0 +1,36 @@
+{% extends "./base_streamfield_block.html" %}
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block block_content %}
+  <section class="tw-bg-white tw-mb-7">
+    <div class="tw-container">
+      <div class="tw-row">
+        <div class="tw-min-h-[400px] medium:tw-mb-7">
+          <div class="tw-relative tw-h-full xlarge:tw-flex tw-justify-end">
+
+          {# Image #}
+            <div class="xlarge:tw-absolute xlarge:tw-left-0 xlarge:tw-w-[65%] xlarge:tw-h-full">
+              {% image value.image fill-928x400 as img %}
+              <img class="tw-h-full tw-object-cover" src="{{ img.url }}" alt="{{ value.alt_text }}">
+            </div>
+
+            {# Content #}
+            <div class="tw-border-b-[4px] tw-border-black tw-top-[2px] tw-self-center tw-my-[0.625rem] tw-bg-white tw-relative  xlarge:tw-w-1/2 xlarge:tw-p-[2.5rem]">
+              {% if value.metadata %}
+                <div class="tw-h6-heading medium:tw-block tw-mb-2">{{ value.metadata }}</div>
+              {% endif %}
+              <h3 class="tw-mb-2 tw-pb-1">
+                {% if value.title_link %}<a href="{% pageurl value.title_link %}">{% endif %}
+                {{ value.title }}
+                {% if value.title_link %}</a>{% endif %}
+              </h3>
```

Comment:
Can we be sure that we have an `h2` before this block? If not, using the `h3` here could lead to invalid document structures.

Reply:
Hmm thats a good question for sure! I've updated this now to be an H2, styled as an H3.

It's does still kind of seem like the same scenario that we can't be sure that this will come before all H3 elements on the page either but I actually think that only matters if there are multiple headings nested under the same parent so H2 should be safer.

Either way I think it's definitely beneficial to have at least some sort of heading that screen readers can use as a landmark haha.  Sorry for the long thought blurb

Reply:
Thanks @stevedya. Yea, I agree, I think `h2` makes it safer. But yes, we are not enforcing editors create a proper semantic structure. 🤔